packages/flutter/test/cupertino/date_picker_test.dart

```diff
@@ -104,6 +105,28 @@ void main() {
       );
     });
 
+    testWidgets('background color default value', (WidgetTester tester) async {
+      final CupertinoTimerPicker picker = CupertinoTimerPicker(
+        onTimerDurationChanged: (Duration value) {},
+      );
+      expect(picker.backgroundColor, Colors.white);
```

Comment:
Thanks for the quick update!

 IMO it makes more sense to verify if the given color was used in painting, something along the lines of:
``` dart
await tester.pumpWidget(picker);
expect(find.byWidget(picker), paints..rect(color: CupertinoColors.white));
```
Didn't really tested it myself so you might need to tweak the code. Also you may need to add `import '../rendering/mock_canvas.dart';` in order to use `paints`.

Reply:
Ah now that I took a quick look into `date_picker.dart`, it might make more sense if you verify 
the `CupertinoPicker` in each column has the right color:
```dart
await tester.pumpWidget(picker);

final Iterable<CupertinoPicker> pickers = tester.allWidgets(find.byType(CupertinoPicker));
expect(pickers.any((CupertinoPicker picker) => picker.backgroundColor != CupertinoColors.white), isFalse);

```